src/components/Router/StepComponentMap.js

```diff
@@ -7,21 +7,28 @@ import {DocumentFrontConfirm, DocumentBackConfrim, FaceConfirm} from '../Confirm
 import Complete from '../Complete'
 import MobileFlow from '../crossDevice/MobileFlow'
 import CrossDeviceLink from '../crossDevice/CrossDeviceLink'
-
+import CrossDeviceClientSuccess from '../crossDevice/ClientSuccess'
```

Comment:
can we just call this client success? I got confused that the name changed implicitly as part of the import